app/models/user_setting.rb

```diff
@@ -7,9 +7,5 @@ def self.default_settings_proc
     }
   end
 
-  validate do
-    unless COLOR_SCHEMES.values.include?(self.color_scheme)
-      errors.add(:color_scheme, "should be one of #{COLOR_SCHEMES.values.to_sentence}")
-    end
-  end
+  validates :color_scheme, :inclusion=> { :in => COLOR_SCHEMES.values }
```

Comment:
Looks like we have removed the informational error message, was it not needed?

Reply:
Seems like there is an option for adding `message` to this.
https://guides.rubyonrails.org/active_record_validations.html#inclusion

Reply:
This comes with a default message that'll say `Color scheme not in list`, don't have to specify one explicitly.